subworkflows/local/fastq_align_rna/main.nf

```diff
@@ -40,8 +44,52 @@ workflow FASTQ_ALIGN_RNA {
     )
     ch_versions = ch_versions.mix(STAR_ALIGN.out.versions.first())
 
+    // Concatenate splice junction files
+    def ch_splice_junctions_to_merge = group_junctions(STAR_ALIGN.out.spl_junc_tab)
+
+    CAT_SPLICE_JUNCTIONS(ch_splice_junctions_to_merge)
```

Comment:
Can we merge the merging/sorting into one process? Seems like a lot of small file shuffling atm